src/JSInterop/Microsoft.JSInterop/src/Infrastructure/ByteArrayJsonConverter.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading;
+
+namespace Microsoft.JSInterop.Infrastructure
+{
+    internal sealed class ByteArrayJsonConverter : JsonConverter<byte[]>
+    {
+        /// Atomically incrementing unique identifier for serializing byte arrays.
+        private static long _byteArrayId;
+
+        internal static readonly JsonEncodedText ByteArrayRefKey = JsonEncodedText.Encode("__byte[]");
+
+        public ByteArrayJsonConverter(JSRuntime jSRuntime)
+        {
+            JSRuntime = jSRuntime;
+        }
+
+        public JSRuntime JSRuntime { get; }
+
+        public override bool CanConvert(Type typeToConvert) =>
```

Comment:
It's super unusual to see ReadOnlySequences appear in interop APIs. Do we really need to support it?

Reply:
Nope not at all, that was a leftover from something I was experimenting. Thanks, removed.